app/gui/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -44,7 +43,7 @@ class Histogram extends Visualization {
 
     constructor(data) {
         super(data)
-        this.setPreprocessor('process_to_json_text', 'Standard.Visualization.Histogram')
+        this.setPreprocessor('Standard.Visualization.Histogram', 'process_to_json_text')
```

Comment:
I like it more. Imagine calling a method by a fully qualified name `Foo.Bar.Baz.quux a b c`. The method just split the call into three sections (arguments are optional in this case)

Reply:
Please double-check that all calls to this have been update. This is a change that can easily cause issues as the arguments have the same type and this will only fail at runtime.